ci-operator/step-registry/workers-rhel/pre-hook/fips/workers-rhel-pre-hook-fips-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: workers-rhel-pre-hook-fips
+  from: ansible
+  commands: workers-rhel-pre-hook-fips-commands.sh
+  resources:
+    requests:
+      cpu: 100m
+      memory: 200Mi
+  env:
+  - name: FIPS_ENABLED
```

Comment:
Actually the `workers-rhel-pre-hook-fips` step name already indicates the step is for enabling fips on rhel workers, maybe we can delete this env.

Reply:
But here it also works for master nodes, could not be deleted.

Reply:
master nodes is not controlled by this step - `workers-rhel-pre-hook-fips`

Reply:
sorry, I didn't check it carefully. The cluster's fips flag is available at https://github.com/openshift/release/blob/a9390d814310021f899a08badd5e925f7e5148ca/ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/workers-rhel8/fips/provision/cucushift-installer-rehearse-gcp-ipi-workers-rhel8-fips-provision-chain.yaml#L34-L37. Yes, it is not mandatory here(at least for fips enabled cluster). Add for fips disabled cluster, it won't  call this ref.

Reply:
Yeah, for provision chain in this PR, the `FIPS_ENABLED` should be set to `true`, while for this ref step, the `FIPS_ENABLED` env var is not needed. If the provision chain call the ref, that means, the cluster enabled fips on rhel workers. So the  `FIPS_ENABLED` env var in this ref, can be deleted, together with https://github.com/openshift/release/blob/a9390d814310021f899a08badd5e925f7e5148ca/ci-operator/step-registry/workers-rhel/pre-hook/fips/workers-rhel-pre-hook-fips-commands.sh#L9-L12